scripts/workers/worker_util.py

```diff
@@ -92,7 +93,7 @@ def update_submission_data(self, data, challenge_pk, submission_pk):
         return response
 
     def update_submission_status(self, data, challenge_pk):
-        url = "/api/jobs/challenge/{}/update_submission/".format(challenge_pk)
+        url = URLS.get("update_submission_status").format(challenge_pk)
```

Comment:
Same as https://github.com/Cloud-CV/EvalAI/pull/2612/files#r365990924

Reply:
Okay, hardcode URL for now.

Reply:
I'm reluctant to hardcode things though.

Reply:
@vkartik97 Removed :+1:

Reply:
> I'm reluctant to hardcode things though.

How about you open an issue for this as you discovered it?

Reply:
That's a good idea. But I'll create it later since it is 5 AM here (I was woken up by earthquake).